pom.xml

```diff
@@ -179,6 +179,7 @@ flexible messaging model and an intuitive client API.</description>
     <rabbitmq-client.version>5.1.1</rabbitmq-client.version>
     <aws-sdk.version>1.11.297</aws-sdk.version>
     <avro.version>1.8.2</avro.version>
+    <joda.version>2.10.1</joda.version>
```

Comment:
We also need to update `distribution/server/src/assemble/LICENSE.bin.txt` with the new version for the dependency or the build will fail.

Reply:
Additionally, I was suggesting to add the dependency here as part of `dependencyManagement` section. This is to force a single version of a particular artifact is used throughout the project

Reply:
> We also need to update distribution/server/src/assemble/LICENSE.bin.txt with the new version for the dependency or the build will fail.

That's actually not a problem here :) because it's a test dependency so it's not included in binary distribution. As general rule though, we try to follow the `dependencyManagement` to pin the deps versions.